.github/workflows/build.yml

```diff
@@ -0,0 +1,103 @@
+name: Build and publish wheels and sdist to PyPI
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      test_pypi:
+        type: boolean
+        description: 'Test release: publish on test.pypi.org'
+        default: false
+
+jobs:
+  build-sdist:
+    name: Build sdist
+    runs-on: ubuntu-latest
+    steps:
+      - name: 💻 Checkout the repo
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+          fetch-tags: true # include tags to get correct version from setuptools_scm
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v6
+
+      - name: 🚀 Install build dependencies
+        run: |
+          pip install build twine
+
+      - name: 📦 Build the sdist and wheel
+        run: python -m build --sdist -o wheelhouse
+
+      - name: List and check sdist
+        run: |
+          ls -lh wheelhouse/
+          twine check wheelhouse/*
+
+      - name: Upload sdist
+        uses: actions/upload-artifact@v4
+        with:
+          name: sdist
+          path: ./wheelhouse/*.tar.gz
+
+  build-wheels:
+    name: 'Build wheels: ${{ matrix.os }}'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
I not sure we should build on windows as anyway I **belive** pyflyby does not work on windows.